rasa/nlu/training_data/formats/rasa.py

```diff
@@ -19,11 +21,11 @@
 
 
 class RasaReader(JsonTrainingDataReader):
-    def read_from_json(self, js, **kwargs) -> "TrainingData":
+    def read_from_json(self, js: Dict[Text, Any], **kwargs) -> "TrainingData":
```

Comment:
Since the `kwargs` are unused
```suggestion
    def read_from_json(self, js: Dict[Text, Any], **_) -> "TrainingData":
```